<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManuInicio</class>
 <widget class="QDialog" name="ManuInicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCCurve</string>
  </property>
  <widget class="QLineEdit" name="edit_user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_password">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ingresar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresar</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_user">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_password">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Contraseña:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_registrar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_password_rep">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Repetir contraseña:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_password_rep">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>171</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/logo4.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_bienvenida">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bienvenid@ :D</string>
   </property>
  </widget>
  <widget class="QLabel" name="inicio_ingresar_oscuro">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inicio_registrar_oscuro">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="inicio_ingresar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inicio_registrar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="espacio_datos">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="espacio_error_inicio">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="text_error_inicio">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>268</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>espacio_datos</zorder>
  <zorder>edit_user</zorder>
  <zorder>edit_password</zorder>
  <zorder>button_ingresar</zorder>
  <zorder>text_user</zorder>
  <zorder>text_password</zorder>
  <zorder>button_registrar</zorder>
  <zorder>text_password_rep</zorder>
  <zorder>edit_password_rep</zorder>
  <zorder>logo</zorder>
  <zorder>text_bienvenida</zorder>
  <zorder>inicio_ingresar_oscuro</zorder>
  <zorder>inicio_registrar_oscuro</zorder>
  <zorder>inicio_ingresar</zorder>
  <zorder>inicio_registrar</zorder>
  <zorder>espacio_error_inicio</zorder>
  <zorder>text_error_inicio</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
